tests/tasks/assert_certificate_parameters.yml

```diff
@@ -5,21 +5,15 @@
 
 - name: Ensure python3 is installed
   package:
-    name:
-      - python3
+    name: python3
 
-- name: Install the package, force upgrade
-  pip:
-    name: pip
-    state: latest  # noqa package-latest
-    virtualenv: "{{ __virtualenv_path }}"
-    virtualenv_command: /usr/bin/python3 -m venv
-
-- name: Install certreader
+- name: Ensure packages are updated
   pip:
     name:
-      - cryptography<35
-      - certreader>=0.1.1
+      - pip
+      - cryptography
+      - pyasn1
+    state: latest  # noqa package-latest
     virtualenv: "{{ __virtualenv_path }}"
     virtualenv_command: /usr/bin/python3 -m venv
```

Comment:
Looks like you need to update `pip` first, by itself, in a separate task:
```yaml+jinja
- name: Update pip
  pip:
    name:
      - pip
    state: latest  # noqa package-latest
    virtualenv: "{{ __virtualenv_path }}"
    virtualenv_command: /usr/bin/python3 -m venv

- name: Ensure packages are installed
  pip:
    name:
      - cryptography
      - pyasn1
    virtualenv: "{{ __virtualenv_path }}"
    virtualenv_command: /usr/bin/python3 -m venv
```

Reply:
Thank you.

Hopefully fixed, as I can't run with rhel-8-9 images right now.